src/Npgsql/TypeHandling/NullableHandler.cs

```diff
@@ -14,11 +14,11 @@ namespace Npgsql.TypeHandling
 
     static class NullableHandler<T>
     {
-        public static readonly Type UnderlyingType;
-        public static readonly ReadDelegate<T> Read;
-        public static readonly ReadAsyncDelegate<T> ReadAsync;
-        public static readonly ValidateAndGetLengthDelegate<T> ValidateAndGetLength;
-        public static readonly WriteAsyncDelegate<T> WriteAsync;
+        public static readonly Type? UnderlyingType;
+        public static readonly ReadDelegate<T>? Read;
```

Comment:
I think this is one of those cases where we shouldn't make these nullable, but have a contract that if Exists returned true these can be called safely. It's better than having to add bangs everywhere where we use it...